### DISCUSSION TITLE
DC Servo Motor position control with an implemented Fuzzy-PID Controller using Raspberry Pi Pico

### DISCUSSION THREAD

**Discussion by Nmsngym** [microsoft/terminal#12426] [Q&A] [answered]
Hello! May I ask if you have any idea on how to program/code a Fuzzy-PID Controller in Raspberry Pi Pico? Or a MATLAB Simulink of the Fuzzy-PID to RPi?
I have a project on implementing a control system, Fuzzy-PID Controller, in my DC Servo Motor for the position control using Raspberry Pi or RPi Pico. Well, my college department does not authorize the use of Arduino. :((
Well, the MATLAB Simulink model is done and already tested, but the implementation on the servo motor is the challenge.

**Comment by zadjii-msft** [comment, marked as answer]
Sorry, I don't really think this is the place to ask for help with your homework.

**Reply by Nmsngym** [reply]
Sorry, this ain't my homework. It is for my design project.

**Reply by zadjii-msft** [reply]
Okay, regardless - we're not really experts in any of the bits of the tech stack you mentioned here. This is the repo for the Windows Terminal and Console. Unless you've got some commandline specific questions, I'm not really sure we're gonna be able to help here 😅

**Reply by Nmsngym** [reply]
Oh alright. It's okay haha. Still, thank you for your reply!
